templates/index.html

```diff
@@ -85,7 +85,7 @@ <h4 class="p-matrix__title"><a class="p-link" href="https://landscape.canonical.
             <img class="p-matrix__img" src="https://assets.ubuntu.com/v1/c5cb0f8e-picto-ubuntu.svg" alt="Icon">
```

Comment:
Please update the link around this image too

Reply:
:ok_hand: